### DISCUSSION TITLE
Spring AI GA release date

### DISCUSSION THREAD

**Discussion by radhakrishna67** [spring-projects/spring-ai#91] [Q&A]
@markpollack , You guys have been doing great contribution for Spring community. We are very curious about Spring AI GA  release. is there any tentative date of GA release?

**Comment by markpollack** [comment]
We got the green light to move the project into the official "spring-projects" repo, so keep an eye out for that this week.   My goal is to release GA very early next year, likely February time frame.  The new repo will have a real roadmap kept in github, unlike this experimental repo.  Thanks for the support.
